### PR TITLE
Inital commit with starter data implemented

### PR DESCRIPTION
**Pull request by JacobWilliams90**

### CODE REVIEW

**Comment by JacobWilliams90** [comment]
Deploy to netlify for stretch 

https://jacobtodoapp.netlify.com/

File: public/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```
Comment (liam-mei) [inline public/index.html:22]: Nice touch with changing the title.  I think it's time to teach you how to make a favicon!

https://formik-material-ui-todo.netlify.com/

If you go to my site, you can see that I changed the favicon to a D for Derrick

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,86 @@
...
+      task: taskName,
+      completed: false
+    };
+
+    this.setState({
+      todo: [...this.state.todo, newItem]
+    });
+  };
+
+  componentWillMount() {
```
Comment (liam-mei) [inline src/App.js:51]: What does componentWillMount do?  What's the difference between that and componentDidMount?

File: src/components/TodoComponents/TodoForm.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    event.preventDefault();
+
+    this.props.addTask(event, this.state.value);
+
+    this.setState({
+      value: ""
+    });
+  };
+
+  handleClear = event => {
```
Comment (liam-mei) [inline src/components/TodoComponents/TodoForm.js:27]: you're passing the event object here, but I don't think we need to at this spot.  I don't think it's doing anything

**Review by liam-mei — APPROVED** [review]

**Comment by liam-mei** [comment]
## MVP
- [x] Input Field
- [x] Submit Button
- [x] Clear All Completed
- [x] Toggle Todos

## Stretch
- [x] Persist Data
- [ ] Search
- [ ] Hosting
- [ ] Styling with Material-UI or other packages
- [x] Use Formik

--------------
score: 3
--------------

**Pull request merged**